### PR TITLE
fixed testcase error of precision

### PR DESCRIPTION
**Pull request by luodanwg**
fixed testcase error of precision

Signed-off-by: luodanwg <luodan.wg@gmail.com>

### CODE REVIEW

File: cli/asset/asset.go [hunk 1]
```diff
@@ -91,7 +91,7 @@ func signTransaction(signer *account.Account, tx *transaction.Transaction) error
 }
 
 func makeRegTransaction(admin, issuer *account.Account, name string, value Fixed64) (string, error) {
-	asset := &Asset{name, byte(0x00), AssetType(Share), UTXO}
+	asset := &Asset{name, byte(0x08), AssetType(Share), UTXO}
```
Comment (dreamfly281) [inline cli/asset/asset.go]: No Magic number, define the  meaning of 0x08 as const

**Comment by luodanwg** [comment]
thanks for review, I have move out the magic number, could you helps to check it again, thanks.

File: core/asset/asset.go [hunk 2]
```diff
@@ -17,6 +17,11 @@ const (
 	Token    AssetType = 0x11
 )
 
+const (
+	MAX_PRECISION = 8
+	MIN_PRECISION = 0
```
Comment (dreamfly281) [inline core/asset/asset.go]: forgive my captious, the underline _ is a good symbol to mark const, but in the golang, especially in our currently coding style, all variable be named with  a-z A-Z  only, the _ is not be encouraged using to keep the style consistent

Comment (luodanwg) [inline core/asset/asset.go]: thanks, i have changed to CamelCase naming rule. could you helps to check it agian thanks.

**Pull request merged**